lib/widget/camera.dart

```diff
@@ -346,17 +359,24 @@ class CameraScreenState extends WidgetState<CameraScreen>
         children: [
           imagePreviewButton(),
           const Spacer(),
-          const Text(
-              'To capture photos and videos, allow access to your camera.'),
+          Text(
+            widget._controller.permissionDeniedMessage ??
+                Utils.translateWithFallback('ensemble.input.maxCountMessage',
+                    'To capture photos and videos, allow access to your camera.'),
```

Comment:
put these messages as const static so you can use them as fallbacks for Utils.getString and also here